torch/_inductor/graph.py

```diff
@@ -2026,9 +2026,15 @@ def compile_to_fn(self) -> Any:
                     serialized_extern_kernel_nodes,
                 )
 
+            additional_files = self.wrapper_code.additional_files
```

Comment:
noob question: when we add strings to generated_files, we will also add the strings to self.wrapper_code.additional_files. Is this a concern?